packages/web-components/src/components/va-minimal-footer/test/va-minimal-footer.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { VAMinimalFooter } from '../va-minimal-footer';
+
+describe('va-minimal-footer', () => {
```

Comment:
Do we need a unit test at all for this component? I think we can accomplish the same thing that you're doing here with an e2e test like this:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-button/test/va-button.e2e.ts#L5-L18

Reply:
@jamigibbs I included the unit test here because the similar e2e test required putting the entire base 64 encoded svg in the test which made it unwieldy.

Reply:
That makes sense, thank you!